app/View/Layouts/default.ctp

```diff
@@ -252,7 +258,7 @@ $ git push</pre>
                   <p>
                   It will also be necessary to update your application to talk to your database back-end.  The <code>config/database.yml</code> file used by rails was set up in such a way that it will accept environment variables for your connection information that you pass to it.
                   Once an administrator has created a MySQL database service for you to connect with you can add the following environment variables to your deploymentConfig to ensure all your frontend pods have access to these environment variables.
-                  
+
 <pre>
 osc env dc/frontend DATABASE_SERVICE_NAME=&lt;database service name&gt;
```

Comment:
I'm not touching this in this PR... but I can do a follow up addressing osc -> oc everywhere (there is already other open PRs doing this)